customer-wallet/src/containers/Customer/CustomerWallet/CustomerWalletForm/CustomerWalletForm.js

```diff
@@ -7,11 +7,39 @@ import { connect } from 'react-redux'
 import { fx } from 'lib/fxRates'
 // components
 import RaisedButton from 'material-ui/RaisedButton'
+import Avatar from 'material-ui/Avatar'
 import Input from 'components/Form/Input'
 import Checkbox from 'components/Form/Checkbox'
 import StarRatingComponent from 'react-star-rating-component'
+import CheckIcon from 'material-ui/svg-icons/toggle/check-box-outline-blank'
+// constants
+import { ratingColor, blue, mainColor, backgroundColorDark, borderColorDark } from 'context/palette'
+
+const submitBtnStyles = {
+  backgroundColor: ratingColor,
+  labelColor: 'rgb(255, 255, 255)'
+}
+
+const avatarStyles = {
+  color: blue,
+  backgroundColor: backgroundColorDark,
+  style: { border: `1px solid ${borderColorDark}` }
+}
+
+const checkboxStyles = (isActive) => ({
+  iconStyle: { fill: isActive ? blue : mainColor }
+})
+
+const textFieldsStyle = {
```

Comment:
@Venumteam I see a fair amount of inline styling used in lots of places in the code.

I think we should move all the inline styles out of js code and put it explicitly in a css file, even if it is a small styling.